grimshot.1

```diff
@@ -116,7 +122,7 @@ using click and drag).\&
 .SH OUTPUT
 .P
 Grimshot will print the filename of the captured screenshot to stdout if called
-with the \fIsave\fR subcommand.\&
+with the \fIsave\fR or \fIsavecopy\fR subcommands.\&
```

Comment:
I don't know why it's rendered like `save  or  savecopy` in `man`. I don't think it's an error. Just curiosity. Anyways, LGTM.